pvr.dvbviewer/addon.xml.in

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.dvbviewer"
-  version="2.0.1"
+  version="3.1.0"
```

Comment:
It's not very important, but you were right. The major bump was  2.0.0 Soy if we do it like the other add-ons, 2.1.0 would be correct here.